tests/installation/partitioning_raid.pm

```diff
@@ -20,6 +20,24 @@ use version_utils qw(is_storage_ng is_sle is_leap is_tumbleweed);
 # tumbleweed is not older product, but it didn't roll out yet
 my $older_product = is_sle('<15') || is_leap('<15.1') || is_tumbleweed;
```

Comment:
I have introduced `is_storage_ng_newui` in `lib/partition_setup.pm`. To be honest I would use just a single place, so when changes reach TW it would be single change instead of multiple places, which we already have. But that's just a side note.

Reply:
Same thing for partition_setup.pm. I can check how to integrate it in subsequent ticket.